test/shared/TestApplicationErrorLogger.cs

```diff
@@ -16,7 +16,7 @@ public class TestApplicationErrorLogger : ILogger
         // Application errors are logged using 13 as the eventId.
         private const int ApplicationErrorEventId = 13;
 
-        public List<Type> IgnoredExceptions { get; } = new List<Type>();
+        public HashSet<Type> IgnoredExceptions { get; } = new HashSet<Type>();
```

Comment:
Is this just for perf? It looks like this collection is only modified at the very start of the test before anything else could even try to read it.

Reply:
Because @pakrym mentioned it